lib/handler.js

```diff
@@ -115,9 +126,16 @@ class Handler {
           summary: description})
 
       console.info({state: status, description: description})
-    }).catch(error => {
-      // (jusx) This should never ever happen. Log it.
-      console.error(error)
+    }).catch(err => {
+      checks.update(
```

Comment:
We should log the error as well so that we are aware of it and can fix the bug.

Reply:
done